Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM openjdk:8-slim
+WORKDIR play
+COPY target/universal/tdr-transfer-frontend*.zip .
+RUN apt-get update && apt-get install unzip && unzip -qq tdr-transfer-frontend-1.0-SNAPSHOT.zip
```

Comment:
The version on line 3 is a wildcard but it's hard-coded on line 4.

Reply:
Well spotted :smile: